### ISSUE TITLE
Unable to type in PS7 window when using Azure Virtual Desktop web client

### ISSUE THREAD

**Issue by Sumtin** [PowerShell/PowerShell#17512]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [x] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

Hi folks, I'm a Cloud admin for a small dev shop and have been running Azure Virtual Desktop (AVD) pools for the last few years to support our developers.  Recently, I received a couple of complaints regarding being unable to type into the PowerShell 7 window.

The problem has been confirmed across multiple Azure tenants and has been repro'd in both Windows 10 and Windows 11 pools.

Steps to reproduce:
1) Open AVD with native Windows client
2) Open PowerShell 7 
3) Start typing (this will work!)
4) Open same AVD pool from web client (this should take over your existing session and kill the native client session)
5) Try typing in the same PowerShell 7 window (this will NOT work)

What does work when using AVD web client:
- Typing in PS7 terminal in VS Code
- Ctrl + C (break) works in PS7 window
- Copy/paste works in PS7 window
- Typing in any other window/app works, including PowerShell 5

### Expected behavior

```console
I see what I type.
```

### Actual behavior

```console
See nothing. :(
```

### Environment data

```powershell
Major  Minor  Patch  PreReleaseLabel BuildLabel
-----  -----  -----  --------------- ----------
7      2      4
```

**Comment by theJasonHelmick** [comment]
Thank you @Sumtin for submitting this issue. I haven;t been able to repo this yet - our internal AVD is very restrictive.  Im in the process of accessing a full customer AVD to test this issue.

**Comment by theJasonHelmick** [comment]
@Sumtin - I spoke with the AVD team, and following their testing suggestions, could not reproduce this error.  My testing environment is not the same as the customers, and AVD recommends that customers experiencing this problem file an issue so they can attempt to repo.

The AVD team has recommended that customers file an issue in Feedback Hub from the AVD host encountering the issue.  Please select the category “Problem”, followed by “Apps” and “Remote Desktop”.  Screenshot below.

<img width="1158" alt="Screen Shot 2022-07-06 at 7 22 33 AM" src="https://user-images.githubusercontent.com/12662278/177576067-83ab82c8-c8bd-48c8-a888-9efcf328fdc0.png">
￼

**Comment by daxian-dbw** [comment]
@Sumtin This was already fixed in the 2.2 version of PSReadLine by https://github.com/PowerShell/PSReadLine/pull/2632.

The v7.2.x of PowerShell ships the v2.1.0 PSReadLine which doesn't have that fix. Please upgrade to [v2.2.6](https://www.powershellgallery.com/packages/PSReadLine/2.2.6) of PSReadLine. See https://github.com/PowerShell/psreadline#upgrading for instructions if you are not familiar with upgrading PSReadLine.

For details about how the root cause is discovered, see the comment thread starting at https://github.com/PowerShell/PowerShell/issues/17783#issuecomment-1211236702.
